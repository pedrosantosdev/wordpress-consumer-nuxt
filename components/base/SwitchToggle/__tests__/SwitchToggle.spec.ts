import { render } from '@testing-library/vue'
import SwitchToggleVue from '@/components/base/SwitchToggle/SwitchToggle.vue'

describe('SwitchToggleVue', () => {
	it('should render properly', () => {
		const value = false
		const wrapper = render(SwitchToggleVue, {
			props: {
				id: 'toggle',
				onToggleSwitch: () => value != value,
			},
		})
		expect(wrapper.html()).toMatchSnapshot()
	})
})
