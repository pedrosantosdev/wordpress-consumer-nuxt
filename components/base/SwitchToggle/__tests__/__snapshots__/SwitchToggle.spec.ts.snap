// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SwitchToggleVue > should render properly 1`] = `"<div data-v-5185cce9=\\"\\" class=\\"relative inline-block w-10 mr-2 align-middle select-none transition duration-200 ease-in\\"><input data-v-5185cce9=\\"\\" id=\\"toggle-input\\" class=\\"toggle-checkbox absolute block w-6 h-6 rounded-full bg-white border-4 appearance-none cursor-pointer\\" type=\\"checkbox\\" name=\\"toggle\\"><label data-v-5185cce9=\\"\\" for=\\"toggle-input\\" class=\\"toggle-label block overflow-hidden h-6 rounded-full bg-gray-300 cursor-pointer\\"></label></div>"`;

exports[`SwitchToggleVue should render properly 1`] = `"<div class="relative inline-block w-10 mr-2 align-middle select-none transition duration-200 ease-in"><input id="toggle-input" class="toggle-checkbox absolute block w-6 h-6 rounded-full bg-white border-4 appearance-none cursor-pointer" type="checkbox" name="toggle"><label for="toggle-input" class="toggle-label block overflow-hidden h-6 rounded-full bg-gray-300 cursor-pointer"></label></div>"`;
